content/blog/2019-02-20-react-translations.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "¡React en español! And Other Languages"
+author: [tesseralis]
+---
+
+We’re excited to announce an ongoing effort to maintain official translations of the React documentation website into different languages. Thanks to the dedicated efforts of React community members from around the world, React is now being translated into *over 30* languages! You can check out our ongoing efforts in the new [Languages](/languages) page!
+
+In addition, the following three languages have completed translating the core pages of the React Docs! 🎉
+
+* Spanish: [es.reactjs.org](https://es.reactjs.org)
+* Japanese: [ja.reactjs.org](https://ja.reactjs.org)
+* Brazilian Portuguese [pt-br.reactjs.org](https://pt-br.reactjs.org)
+
+## Why Localization Matters
+
+React is one of the most powerful JavaScript libraries and has [meetups](https://reactjs.org/community/meetups.html) and [conferences](https://reactjs.org/community/conferences.html) all around the world. Not all of the places where React is used has English as a primary language. We'd love to support local communities who use React, and maintaining official translations is one way to do that!
+
+In addition, if there is enough demand in a language community, people *will* create translations for it, as people have done in [Chinese](https://github.com/discountry/react), [Arabic](https://wiki.hsoub.com/React), and [Korean](https://github.com/reactjs/ko.reactjs.org/issues/4). While useful, these resources are also hard to find. With an official translation, people can find the translated docs from the React website itself and translators have access to the React team to clarify any questions.
```

Comment:
This section almost reads as defensive to me. What do you think of this angle:

---

React already has many meetups and conferences around the world, but many programmers don't use English as their primary language. We’d love to support local communities who use React by making our documentation available in most popular languages.

In the past, React community members have created unofficial translations for Chinese, Arabic, and Korean; by making an official channel for these translated docs we're hoping to make them easier to find and help make sure that non-English-speaking users of React aren't left behind.